python/bin/repl_stub.py

```diff
@@ -63,8 +63,7 @@ def complete(self, text, state):
         readline.parse_and_bind("tab: complete")
     else:
         print(
-            "Could not enable tab completion: "
-            "unable to determine readline backend"
+            "Could not enable tab completion: " "unable to determine readline backend"
```

Comment:
Nit: merge the strings

```suggestion
            "Could not enable tab completion: unable to determine readline backend"
```